<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New</class>
 <widget class="QWidget" name="New">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="tower1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>0</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tower3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tower2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="levelup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>‚¨ÜÔ∏è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inform">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>üîô</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pause">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>51</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>üîô</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
